### PR TITLE
Backup the directory where RabbitMQ definitions are exported

### PR DESCRIPTION
**Pull request by bdero**
CC @smarnach 

This change goes along with https://github.com/open-craft/ansible-rabbitmq/pull/6

### CODE REVIEW

File: group_vars/rabbitmq/public.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ SANITY_CHECK_LIVE_PORTS:
     port: 15671
     message: "RabbitMQ management does not respond on port 15671."
 
-TARSNAP_BACKUP_FOLDERS: /etc/rabbitmq /var/lib/rabbitmq /etc/letsencrypt
+TARSNAP_BACKUP_FOLDERS: /usr/local/lib/rabbitmq/backup /etc/rabbitmq /etc/letsencrypt
```
Comment (smarnach) [inline group_vars/rabbitmq/public.yml]: I think we can simply use `{{ RABBITMQ_EXPORT_DIR }}` as the first directory here, to guarantee it matches.

Comment (bdero) [inline group_vars/rabbitmq/public.yml]: I'm surprised that the scoping works out this way. I thought that it wouldn't due to stuff like `RABBITMQ_EXPORT_DIR` only being defined in the `rabbitmq` role and `TARSNAP_BACKUP_FOLDERS` being used in the tarsnap backup role, but it works. Thanks!

**Comment by bdero** [comment]
@smarnach Addressed your comment!

File: group_vars/rabbitmq/public.yml (hunk 1 above)
Comment (smarnach) [inline group_vars/rabbitmq/public.yml]: I think "scoping" is the wrong word to describe how Ansible resolves variables.  There is only one scope, and everything is dumped into that single scope (in a [defined order](http://docs.ansible.com/ansible/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable)) before any inline Jinja2 templates are rendered.  This process results in a whole lot of, well, unexpected behaviour, to put it friendly, e.g. if you conditionally include a role and the condition is false, you will see its variables anyway.  But at least it has the convenient effect that you can see all variables everywhere, and they have the same value everywhere (until you change them with `set_fact`, that is).

**Comment by smarnach** [comment]
:+1: Looks good to me!

- [x] ~I tested this: (describe what you tested)~ Let's test this by deploying it to production.
- [x] I read through the code
- [x] ~I checked for accessibility issues~
- [x] ~Includes documentation~

**Pull request merged**